examples/project-quota/README.md

```diff
@@ -109,20 +114,24 @@ $ cat pod-without-resources.yaml
 apiVersion: v1
 kind: Pod
 metadata:
+  creationTimestamp: null
   name: pod-without-resources
 spec:
   containers:
   - capabilities: {}
     image: gcr.io/google_containers/serve_hostname
     imagePullPolicy: IfNotPresent
     name: kubernetes-serve-hostname
+    resources: {}
     securityContext:
       capabilities: {}
       privileged: false
+    terminationMessagePath: /dev/termination-log
   dnsPolicy: ClusterFirst
   restartPolicy: Always
+status: {} 
 $ oc create -f pod-without-resources.yaml
-Error from server: Pod "pod-without-resources" is forbidden: Limited to 750Mi memory, but pod has no specified memory limit
+Error from server: error when creating "pod-without-resources.yaml": pods "pod-without-resources" is forbidden: Failed quota: quota: must specify cpu,memory
```

Comment:
There is a stutter if the quota name is quota.  I will look at what I can do to improve it upstream.